pom.xml

```diff
@@ -240,6 +240,7 @@
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
                 <version>${mysql.version}</version>
+                <scope>test</scope>
```

Comment:
We should add this to the docs to tell users downloading mysql jar locally.

Reply:
There are instructions in the deployment document (https://streampark.apache.org/docs/platform/platform-build#introduce-mysql-dependency-package). At the same time, the instructions will be checked when starting the project (https://github.com/apache/incubator-streampark/blob/f9421f6bbc162200cd9accdc25ac2bd43e78e889/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/SpringProperties.java#L56-L69)

Reply:
I see. Thanks for clarification.